src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -32,51 +84,234 @@ const ContactUs = () => {
       timestamp: new Date().toLocaleString('en-US', { timeZone: 'America/Los_Angeles' }),
       been_dismissed: false,
     };
-    const result = await backend.post('/notification/', updatedFormData);
+    await backend.post('/notification/', updatedFormData);
+    setShowConfirmation(true);
   };
 
-  const handleInputChange = e => {
-    let inputValue = e.target.value;
-    setText(inputValue);
+  const handleNumInputChange = (e, index) => {
+    checkedItems[index] = Number(e);
+
+    // Update the state with the new array
+    setCheckedItems(checkedItems);
+    isFormFilled();
+    setIsFormValid(checkedItems.some(value => value !== 0) || text.length > 0);
+  };
+
+  const isFormFilled = () => {
+    // Check if all form fields are filled
+    return checkedItems.some(value => value !== 0) || text.length > 0;
   };
 
   const setMessage = text => {
     const textCopy = text;
-    const checkedThingies = ['Stickers', 'Posters', 'Business Cards', 'Something', 'Joshua Lipton'];
-    const preMessage = `${name} is requesting: \n`; // business name... hard-coded right now
+    const checkedThingies = [
+      'DS Decals',
+      'Rack Cards',
+      'Donation Envelopes',
+      'Business Cards',
+      'Stickers',
+      'Bin Decals',
+      'DS Decals',
+    ];
+    const preMessage = `Business is requesting: \n`; // business name... hard-coded right now
     const updatedMessage = checkedThingies.filter((_, index) => checkedItems[index]).join('\n-');
     return `${preMessage}-${updatedMessage}\nNotes:\n${textCopy}`;
   };
```

Comment:
Check the comment above ^^, make sure to add the number of items and insert it into the string.

Reply:
![image](https://github.com/ctc-uci/fph-frontend/assets/25378073/63909a02-6636-40f8-a9eb-679925715ab1)